build/yamls/antrea.yml

```diff
@@ -1376,6 +1376,17 @@ data:
     # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
     # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
     #kubeAPIServerOverride: ""
+
+    # Comma-separated list of cipher suites. If omitted, the default Go cipher suites will be used.
+    # https://golang.org/pkg/crypto/tls/#pkg-constants
+    # Note that Antrea is using an old version of k8s.io/component-base, so TLS1.3 cipher suites cannot be explicitly listed here.
+    # Additionally, by default Antrea uses the Go "crypto" libraries, which do not support configuring Cipher Suites for TLS1.3.
+    # In other words, the apiserver will always prefer TLS1.3 Cipher Suites whenever possible.
```

Comment:
I do not understand this sentence - why "in other words", "the apiserver will always prefer TLS1.3 Cipher Suites whenever possible"? I cannot see the previous sentences are relevant to "the apiserver will always prefer TLS1.3 Cipher Suites whenever possible".

Reply:
https://github.com/golang/go/issues/29349

You cannot disable TLS1.3 Cipher Suites and they will always be preferred to the cipher suites you explicitly provide. So if you set `ltsCipherSuites` to `<some TLS1.2 suite>`, your choice will be ignored if the client supports TLS 1.3

Reply:
Should we just explicitly declare the two points: TLS1.3 cipher suites cannot be listed; but apiserver will always prefer TLS1.3 when it is supported by the client?

Reply:
I think @jianjuns 's concern is that "in other words" isn't proper here. I changed it to "actually". Do you think it is better?